From e784b9b6d11c5a2a1307f936e3dbf0435bd568e2 Mon Sep 17 00:00:00 2001
From: George Mocanu <george.mocanu@nxp.com>
Date: Fri, 6 Dec 2024 14:15:46 +0200
Subject: [PATCH 3/3] fdts: s32g: Disable serdes1 node in PFE Slave DT

Serdes1 is managed/configured by the real-time applications
when the Linux domain acts as the PFE Slave.

Signed-off-by: George Mocanu <george.mocanu@nxp.com>
---
 fdts/s32g-goldvip-pfe-slave.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/fdts/s32g-goldvip-pfe-slave.dtsi b/fdts/s32g-goldvip-pfe-slave.dtsi
index deba12b65..12c991803 100644
--- a/fdts/s32g-goldvip-pfe-slave.dtsi
+++ b/fdts/s32g-goldvip-pfe-slave.dtsi
@@ -6,3 +6,7 @@
 &qspi {
 	status = "okay";
 };
+
+&serdes1 {
+	status = "disabled";
+};
-- 
2.34.1

